sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -252,6 +216,42 @@ export enum ReceiveMode {
     receiveAndDelete = 2
 }
 
+// @public
+export interface Receiver<ContextT> {
+    close(): Promise<void>;
+    diagnostics: {
+        peek(maxMessageCount?: number): Promise<ReceivedMessage[]>;
+        peekBySequenceNumber(fromSequenceNumber: Long_2, maxMessageCount?: number): Promise<ReceivedMessage[]>;
+    };
+    // (undocumented)
+    entityPath: string;
+    // (undocumented)
+    entityType: "queue" | "subscription";
+    // (undocumented)
+    getDeadLetterPath(): string;
+    getMessageIterator(options?: GetMessageIteratorOptions): AsyncIterableIterator<{
+        message: ReceivedMessage;
+        context: ContextT;
+    }>;
+    // (undocumented)
+    isReceivingMessages(): boolean;
+    receiveBatch(maxMessages: number, maxWaitTimeInSeconds?: number, options?: ReceiveBatchOptions): Promise<{
+        messages: ReceivedMessage[];
+        context: ContextT;
+    }>;
+    // Warning: (ae-forgotten-export) The symbol "Long" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    receiveDeferredMessage(sequenceNumber: Long_2, options?: OperationOptions): Promise<ServiceBusMessage | undefined>;
```

Comment:
Why `Long_2`?

Reply:
`import Long from "long";` in the src files should fix this.

Reply:
Indeed. Echoing what @chradek has said before - I'm still unclear why this never produces a compile error.